components/input/ColorPicker.js

```diff
@@ -1,33 +1,57 @@
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { noop } from 'proton-shared/lib/helpers/function';
-import { Icon, DropdownButton } from 'react-components';
-import { ChromePicker } from 'react-color';
+import { Icon, Dropdown, DropdownButton, generateUID, usePopperAnchor, LinkButton } from 'react-components';
 import tinycolor from 'tinycolor2';
 
-import './ColorPicker.scss';
+const COLORS = [
+    '#d83078',
+    '#1ac6db',
+    '#db6b15',
+    '#a0db13',
+    '#30304d',
+    '#33908f',
+    '#fe5f55',
+    '#f5d85d',
+    '#a5cbc3',
+    '#96c5f7'
+];
 
-const ColorPicker = ({ color = 'blue', onChange = noop, ...rest }) => {
-    const [display, setDisplay] = useState(false);
+const ColorPicker = ({ color = 'blue', onChange = noop }) => {
     const colorModel = tinycolor(color);
     const iconColor = colorModel.isValid() ? colorModel.toHexString() : '';
-    const handleClick = () => setDisplay(!display);
-    const handleClose = () => setDisplay(false);
 
-    const picker = (
-        <div className="popover">
-            <div className="cover" onClick={handleClose} />
-            <ChromePicker color={color} onChange={onChange} />
-        </div>
-    );
+    const [uid] = useState(generateUID('dropdown'));
+    const { anchorRef, isOpen, toggle, close } = usePopperAnchor();
+
+    const handleClick = (color) => () => onChange({ hex: color });
 
     return (
-        <div className="relative">
-            <DropdownButton onClick={handleClick} hasCaret={true} {...rest}>
+        <>
+            <DropdownButton buttonRef={anchorRef} isOpen={isOpen} onClick={toggle} hasCaret={true}>
                 <Icon className="flex-item-noshrink" name="circle" color={iconColor} />
```

Comment:
Maybe add some text here using `sr-only` class (like "Choose color" or "choose color (blue selected)", otherwise it will say "button".